neurolib/models/multimodel/builder/base/neural_mass.py

```diff
@@ -0,0 +1,204 @@
+"""
+Base for all mass models.
+"""
+import symengine as se
+from jitcdde import y as state_vector
+
+
+class NeuralMass:
+    """
+    Represent a neural population with given parameters and equations, in
+    particular, the derivatives of the state vector.
+    """
+
+    name = ""
+    label = ""
+
+    # type of the mass - usually excitatory or inhibitory,
```

Comment:
so... I was thinking a lot about this but I believe that the easiest solutions are the best ones (usually).. so I added assert that when constructing `Node` with more masses than 1, the mass_types need to be unique!
so, typical examples:
* basic EXC-INH node: one `mass_type` is `EXC`, other is `INH`
* some W-C extensions like the one on PR here: `EXC`, `SST+` and `PV+`
* e.g. layer-resolving AdEx could have `layer-1-EXC`, `layer-1-INH`, etc...

this way, all the results will be consistent and it's future proof - you can have thousands of masses in the node, as long as each has unique `mass_type`.. of course for thousands of masses in one node, you can do this programmatically :)

Reply:
Perfect, I think that's a great solution!